python/cudf/cudf/tests/test_replace.py

```diff
@@ -364,6 +364,37 @@ def test_series_with_nulls_where(fill_value):
     assert_eq(expect, got)
 
 
+@pytest.mark.parametrize("fill_value", [[888, 999]])
```

Comment:
Can we parametrize on dtypes as well? Would ideally want to test category, string, float, and int